packages/rollup-plugin-import-meta-assets/src/rollup-plugin-import-meta-assets.js

```diff
@@ -100,10 +100,12 @@ function importMetaAssets({ include, exclude, warnOnError, transform } = {}) {
               );
               modifiedCode = true;
             } catch (error) {
-              if (warnOnError) {
-                this.warn(error, node.arguments[0].start);
-              } else {
-                this.error(error, node.arguments[0].start);
+              if (error.code !== 'EISDIR') {
+                if (warnOnError) {
+                  this.warn(error, node.arguments[0].start);
+                } else {
+                  this.error(error, node.arguments[0].start);
+                }
```

Comment:
I think we should instead match only expressions which contain an extension in the path by patching `isNewUrlImportMetaUrl` method, so that we don't even run into this error

also maybe good to rename `isNewUrlImportMetaUrl ` to `isNewUrlAssetUrl`

Reply:
[this comment](https://github.com/modernweb-dev/web/issues/2421#issuecomment-1690682097) address this, so I'm OK with the current check for `EISDIR`